src/http_server.cc

```diff
@@ -2173,6 +2173,12 @@ HTTPAPIServer::EVBufferToInput(
     }
   }
 
+  // get real header length
+  if (n > 0 && typeid(*this) != typeid(HTTPAPIServer)) {  // only for adsbrain
+    header_length = *(unsigned int *)((char *)v[n-1].iov_base + v[n-1].iov_len - 4);
```

Comment:
What does `4` refer to here?

Reply:
sizeof(unsigned int), the additional value appended.